codes/javascript/chapter_computational_complexity/leetcode_two_sum.js

```diff
@@ -2,7 +2,7 @@
  * @Author: gyt95 (gytkwan@gmail.com)
  * @Date: 2022-12-15 10:51:54 
  * @Last Modified by: gyt95 (gytkwan@gmail.com)
```

Comment:
Please set the file header following the format of other JS files.

Reply:
I think it should work this time.

Reply:
I forgot something in test, wait a minute.